Infragraph LA docs update [IPE-1505] (#2710)
hashicorp/web-unified-docs: content/hcp-docs/content/docs/infragraph/connections/aws.mdx

  The AWS connection connects your Amazon Web Services data to the resource graph. Learn how to set up the connection, sync data, and view AWS data in Infragraph.
---

# Connect Infragraph to AWS

The AWS connection connects your Amazon Web Services data to the resource graph. 

## Set up the connection 

Complete the following steps to establish a new AWS connection:

1. From the Resource graph **Overview** at the top right, click **Create Connection**. 
1. Select **Amazon Web Services (AWS)** from the list of data sources. 
1. Click **Select AWS**.
1. Click **View Terraform module in Registry** to review the IAM Role OIDC Federation and IAM Policy requirements scope.
1. Copy the generated module block and add it to your HCP Terraform or Terraform Enterprise linked VCS repository.
    1. Configure your AWS credentials for the accounts you added in the sync scope. Refer to the [AWS Terraform provider authentication and configuration](https://registry.terraform.io/providers/hashicorp/aws/latest/docs#authentication-and-configuration) documentation for more information.
    1. Run a Terraform apply operation to create the required IAM resources in your AWS environment.

> 1. We recommend subscribing to the module GitHub repository to be notified of changes to ensure your connections continue to work.

Reviewer: This step is not directive (but it's also a weird step in the process, with no checkbox or anything).

Maybe "We recommend clicking the **module repo** link and subscribing to the module repository on GitHub so you will be notified..."?

1. Click the **Provide connection details** button.
1. Choose a **Sync scope**:
    - **Organization**: Infragraph syncs all accounts in the organization. Enter the **Management account ID** for the organization.

>     - **Specific Accounts**: Infragraph only syncs data from the specified accounts. Enter the **AWS Account IDs** you want to include in this connection. You can enter IDs manually separated by commas or new lines or click **Upload CSV** to import up to 100 accounts in bulk.

Reviewer: ```suggestion
    - **Specific Accounts**: Infragraph only syncs data from the specified accounts. Enter the **AWS Account IDs** you want to include in this connection. You can manually enter IDs separated by commas or new lines or click **Upload CSV** to import up to 100 accounts in bulk.
```

1. Click the **Create connection** button.

## Sync AWS data 

@include 'infragraph/sync-data.mdx'

## View AWS data  

To find AWS data connected to your resource graph, complete any of the following actions:

- Go to the **Inventory** page. Click an individual resource type to view it in the graph explorer. 

> - Query resources with any of the existing queries that include AWS resources. 

Reviewer: ```suggestion
- Query resources with a saved query that includes AWS resources. 
```
